apps/frontend/src/network/teams/__tests__/About.test.tsx

```diff
@@ -70,3 +75,20 @@ describe('the proposal', () => {
     expect(getByText(/proposal/i).closest('a')!.href).toMatch(/someproposal$/);
   });
 });
+
+it('renders the team groups card when a group is present', async () => {
+  mockedGetTeamGroups.mockResolvedValue({
+    items: [
+      {
+        ...createGroupResponse(),
+        name: 'Example Group 123',
+      },
+    ],
+    total: 1,
+  });
+  const { getByText } = await renderTeamAbout({
+    team: { ...createTeamResponse(), proposalURL: undefined },
+  });
+  await waitFor(() => expect(mockedGetTeamGroups).toHaveBeenCalled());
```

Comment:
using `await findByText` instead of `getByText` avoids an explicit wait

Reply:
I couldn't get this to work. I don't think its worth fighting with